.jenkins/pytorch/test.sh

```diff
@@ -407,9 +407,21 @@ test_torch_function_benchmark() {
   assert_git_not_dirty
 }
 
+build_xla() {
+  clone_pytorch_xla
+  # shellcheck disable=SC1091
+  source "xla/.circleci/common.sh"
+  apply_patches
+  install_deps_pytorch_xla xla
+  set_torch_cmake_prefix_path
```

Comment:
Surprisingly `xla/.circleci/common.sh` also decides to muck around with `CMAKE_PREFIX_PATH` so we need to set it after we source `xla/.circleci/common.sh`

Reply:
Submitted https://github.com/pytorch/xla/pull/3106 to fix